charters/wot-ig-2019.html

```diff
@@ -226,7 +226,9 @@ <h2>Scope</h2>
           The Web of Things Interest Group (WoT IG) is a forum for discussion of ideas relating to the Web of Things
           and is intended to complement the role of the Web of Things Working Group (WoT WG).
           In particular, the WoT IG will propose new work items based upon new requirements, use cases,
-          and discussions with IoT developer communities, industry alliances, and standards development organizations.
+          and discussions with IoT developer communities, industry alliances,
+          vertical industry solution and infrastructure providers,
```

Comment:
I would change the order a bit and put SDOs and industry alliances higher in the rank.

Reply:
how about putting SDOs first, then industry alliances and then providers.   then SDOs are grouped with industry alliances and alliances are also grouped with providers.